examples/ct/ct_handlers.go

```diff
@@ -44,23 +62,40 @@ type addChainRequest struct {
 	Chain []string
 }
 
+type addChainResponse struct {
+	SctVersion int    `json:sct_version`
+	ID         string `json:id`
+	Timestamp  string `json:timestamp`
```

Comment:
JSON works with floats, this value is outside the range that can be represented. Or is there a trick to this somewhere?